<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrganizationPersonsWindow</class>
 <widget class="QMainWindow" name="OrganizationPersonsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="org_persons_tree_widget">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Username</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Org ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>is Org Admin</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="search_by_label">
         <property name="text">
          <string>Search by:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="search_by_name_radio">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="search_by_username_radio">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="search_by_org_Id_radio">
         <property name="text">
          <string>Org ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="search_line_edit">
         <property name="placeholderText">
          <string>Name, Username or Org ID...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="members_checkbox">
         <property name="text">
          <string>Members</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Admins_checkbox">
         <property name="text">
          <string>Admins</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
